Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+buildPlugin(
+  useContainerAgent: true,
+  configurations: [
+    [platform: 'linux', jdk: 11]
```

Comment:
Deliberately skipping Windows, and Java 17/21?

Reply:
for now yes, there is no code where I see the need to run on windows or use a newer jdk (maybe one can enable jdk21 though)